semanticdb/metap/src/main/scala/scala/meta/cli/Metap.scala

```diff
@@ -0,0 +1,221 @@
+package scala.meta.cli
+
+import java.io._
+import java.util._
+import scala.collection.mutable
+import scala.compat.Platform.EOL
+import scala.math.Ordering
+import scala.util.control.NonFatal
+import scala.meta.internal.semanticdb3._
+import SymbolInformation._, Kind._, Property._
+import SymbolOccurrence._, Role._
+import Diagnostic._, Severity._
+
+object Metap {
+  def main(args: Array[String]): Unit = {
+    sys.exit(process(args))
+  }
+
+  def process(args: Array[String]): Int = {
```

Comment:
Pro tip, accept a `PrintStream` and this gets much easier to test ;)

Reply:
👍

Reply:
On a second thought, the required changes seem to be quite extensive. I suggest we keep things as is.